terraform/environments/core-network-services/wanted-firewalls.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Will discuss these. Initial layout and settings so I will amend the source and destination ports. Current code is copied from terraform so it will require some amendments